assets/js/buttons-examples.js

```diff
@@ -104,6 +104,12 @@ $(document).ready(function () {
           text: "TopRight-BottomLeft",
           shouldHaveRoundedType: false,
         },
+        {
```

Comment:
Place this in the correct alphabetical order under "Button Styles" and change label to just "Stack"

Reply:
Got it !✌🏻